psw-backend/src/main/java/com/example/pswbackend/serviceImpl/AppointmentServiceImpl.java

```diff
@@ -313,6 +313,7 @@ public void sendOperationMail(Appointment appointment) {
     }
 
     @Override
+    //@Transactional(readOnly = false, propagation = Propagation.REQUIRES_NEW)
```

Comment:
da bi ti ovo radilo trebas da stavis iznad klase @trans = true

Reply:
Inace ti izbacuje da je readonly operacija

Reply:
jaoooooooo stvarno, cekaj samo da vidim, to me je mucilo